src/main/java/seedu/address/ui/StudentCard.java

```diff
@@ -19,8 +17,9 @@
  */
 public class StudentCard extends UiPart<Region> {
 
-    private static final List<Tag> listOfTags = new ArrayList<>();
-    private static final List<Color> listOfColors = new ArrayList<>();
+    private static final List<Tag> TAG_LIST = new ArrayList<>();
```

Comment:
I think we should retain the name `listOfTags` or `tagList` instead? Although the field was declared as `static final`, it is not exactly a constant in essence since the contents of the list can change.

I think it would be wise to follow the following categorisation of constants as given by the excerpt below from the Google Style Guide.
![Screenshot 2020-10-05 at 10 07 05 PM](https://user-images.githubusercontent.com/1428635/95089781-1c1ace00-0757-11eb-9431-9e89e67904ae.png)

Reply:
I agree with Dexter on this